src/usr/local/www/bootstrap/css/pfSense-dark.css

```diff
@@ -157,6 +157,11 @@ ul.context-links li a {
     border-color: #212121;
 }
 
+.btn-breadcrumbbar {
+	color: #808080;
+    background-color: #404040;
+}
```

Comment:
I'm not sure it is necessary to change the background color here, or if it is it should be the same color as the breadcrumb background so that there does not appear to be a rectangular box around the text.

Also consider adding a cursor statement here to that the cursor changes to a pointer when hovering over the control.